examples/crypto.toit

```diff
@@ -0,0 +1,204 @@
+
+import crypto.aes
+import expect
+
+
+/**
+Randomly generated key with 256 bits.
+Generated from https://www.allkeysgenerator.com/Random/Security-Encryption-Key-Generator.aspx.
+*/
+secret32 ::= #[
+    0x45, 0x29, 0x48, 0x2B, 0x4D, 0x62, 0x51, 0x65, 
+    0x54, 0x68, 0x57, 0x6D, 0x5A, 0x71, 0x34, 0x74,
+    0x37, 0x77, 0x21, 0x7A, 0x25, 0x43, 0x2A, 0x46,
+    0x2D, 0x4A, 0x61, 0x4E, 0x63, 0x52, 0x66, 0x55
+]
+
+//Use the lowest 192 bits of the 256 bit key
```

Comment:
We typically have a space after // and terminate the comments with .

Reply:
Fixed for all comments.